setup.py

```diff
@@ -53,8 +53,7 @@
 from os import path, walk
 from io import StringIO
 
-# Note: cocotb is not installed properly yet and is missing dependencies and binaries
-# We can still import other files next to setup.py, as long as they're in MANIFEST.in
```

Comment:
I think this comment is still mostly accurate, despite the new path code.

Reply:
Added it back and added a note about the sys.path line.